index.js

```diff
@@ -36,21 +64,21 @@ const defaultTarget = '_blank'
 const defaultRel = ['nofollow', 'noopener', 'noreferrer']
 const defaultProtocols = ['http', 'https']
 
+/**
+ *
+ * @param {Options[keyof Options]} value
+ * @param {Element} node
+ */
+function createFunction(value, node) {
```

Comment:
This name to me implies that it would instead always return a function, from an arbitrary value.
The inverse is true. Maybe a name such as `callOrGet`, `callIfNeeded`, `callFunction`?